docs/spec/nmt.md

```diff
@@ -143,6 +143,36 @@ Note that hash of the leaf does not have to be verified against the underlying m
 
 If the queried `nID` falls outside the namespace range of the tree root, or the namespace tree is empty, then an empty NMT proof is valid by definition.
 
+### Index-based Merkle Range Proof
```

Comment:
fmt: could you run https://github.com/DavidAnson/markdownlint with fixes on this?

Reply:
Sure, the issues are fixed now https://github.com/celestiaorg/nmt/pull/162/commits/218d517480d869eec94f28ea90633925a9e22439